### PR TITLE
Add amp-ad support for Red for Publishers

### PR DESCRIPTION
**Pull request by spiritloose**
Adding Red for Publishers ad network to the list of supported ad networks.

### CODE REVIEW

File: ads/rfp.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+</amp-ad>
+```
+
+## Configuration
+
+For configuration details and to generate your tags, please contact https://www.fout.co.jp/freakout/contact/.
+
+Supported parameters:
+
+- `data-adspot-id`
```
Comment (zhouyx) [inline ads/rfp.md]: From your code, looks like `data-adspot-id` is required, while `data-sytlesheet-url` and data-country` are optional. Please indicate. You can refer to our [example doc](https://github.com/ampproject/amphtml/blob/master/ads/_ping_.md#required-parameters).

File: ads/rfp.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<!---
```
Comment (zhouyx) [inline ads/rfp.md]: Please add your link to your readme doc from the ad readme doc. https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad/amp-ad.md#supported-ad-networks

File: examples/ads.amp.html [hunk 3]
```diff
@@ -1420,6 +1421,12 @@ <h2>RevJet Tag</h2>
       data-opts='{"delivery_method": "banner"}'>
   </amp-ad>
 
+  <h2>Red for Publishers</h2>
+  <amp-ad width="320" height="150"
```
Comment (zhouyx) [inline examples/ads.amp.html]: I found this ad slot always return me a no fill ad. Could you please provide a fake ad example here, it's easier for us to test your code, and also easier to locate possible regression in the future. Thanks.

File: ads/rfp.md (hunk 2 above)
Comment (spiritloose) [reply ads/rfp.md]: Added by 4397718

File: ads/rfp.md (hunk 1 above)
Comment (spiritloose) [reply ads/rfp.md]: Fixed by fbfbfbe

File: examples/ads.amp.html (hunk 3 above)
Comment (spiritloose) [reply examples/ads.amp.html]: I set it to always be displayed.
Please note that in order to display it, it's necessary to set browser as mobile User-Agent.

**Comment by spiritloose** [comment]
@zhouyx Some conflicts resolved in 5ac865c .

**Comment by nainar** [comment]
Pinging @zhouyx for a review.

**Comment by zhouyx** [comment]
Sorry for the delay. Looks great. LGTMed an merged.

**Pull request merged**